src/helpers/voice/join_voice_channel.ts

```diff
@@ -0,0 +1,19 @@
+import { DiscordGatewayOpcodes } from "../../types/codes/gateway_opcodes.ts";
+import type { UpdateVoiceState } from "../../types/voice/update_voice_state.ts";
+import { requireBotChannelPermissions } from "../../util/permissions.ts";
+import { sendShardMessage } from "../../ws/send_shard_message.ts";
+import { calculateShardId } from "../../util/calculate_shard_id.ts";
+import { snakelize } from "../../util/utils.ts";
+
+export async function connectToVoiceChannel(
+  guildId: bigint,
+  channelId: bigint,
+  options?: Omit<UpdateVoiceState, "guildId" | "channelId">
```

Comment:
Need to be partial else users have to pass both. Or create an at least one type.

Reply:
Why was this resolved?
The type has to be partial.